Bottom: 605530dae4f4f7a6a4cd2210003feba632354ac7
Top:    a582d774a18656e756c9add470227636876036c4
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-12 23:54:58 +0000

Refresh of hidgl-fixup-to-use-hid_draw

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index c594dca..b8409e3 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -293,7 +293,7 @@ hidgl_set_depth (hidGC gc, float depth)
 }
 
 void
-hidgl_draw_grid (hidGC gc)
+hidgl_draw_grid (hidGC gc, BoxType *drawn_area)
 {
   hidglGC hidgl_gc = (hidglGC)gc;
   HID_DRAW *hid_draw = gc->hid_draw;
@@ -998,7 +998,7 @@ fill_polyarea (hidGC gc, POLYAREA *pa, const BoxType *clip_box, bool use_new_ste
 }
 
 void
-hidgl_fill_pcb_polygon (hidGC gc, PolygonType *poly, const BoxType *clip_box))
+hidgl_fill_pcb_polygon (hidGC gc, PolygonType *poly, const BoxType *clip_box)
 {
   bool use_new_stencil;
